arch/arm/soc/st_stm32/Kconfig

```diff
@@ -9,6 +9,7 @@ config SOC_FAMILY_STM32
 	bool
```

Comment:
Could you change the commit message to: "Enable HAS_SEGGER_RTT to allow user to select use of SEGGER RTT flash and debug solution"

Reply:
en, HAS_SEGGER_RTT doesn't select a flash and debug solution, it only select SEGGER RTT. so I think this is better:
Enable HAS_SEGGER_RTT to allow user to select use of SEGGER RTT and SystemView solution.

What's your idea?